commands/usernote.js

```diff
@@ -70,11 +70,11 @@ module.exports = {
     });
     let parseSettings = {
       default: {type: 'yellow'},
-      string: ['subreddit', 'note', 'user', 'type', 'link', 'refresh'],
-      boolean: ['refresh', 'help'],
+      string: ['subreddit', 'note', 'user', 'type', 'link', 'refresh', 'all'],
+      boolean: ['refresh', 'help', 'all'],
```

Comment:
I think `all` should just get added to `boolean` here, not `boolean` and `string`.